test/e2e/Makefile

```diff
@@ -51,9 +55,18 @@ $(E2E_DATA_DIR): $(TMP_CAPV_DIR)
 	$(MAKE) -C $(TMP_CAPV_DIR) e2e-templates && cp -r $(TMP_CAPV_DIR)/test/e2e/data $(E2E_DATA_DIR) && \
 	cp $(TMP_CAPV_DIR)/metadata.yaml $(E2E_DATA_DIR)
 
-run: $(TOOLING_BINARIES) $(E2E_DATA_DIR)
+push-ccm-image:
```

Comment:
Does all e2e test use the same image tag? Then they could conflict with each other. How does capv test their dev image